tailwind.config.cjs

```diff
@@ -76,6 +77,23 @@ module.exports = withMT({
       xl: "1280px",
       "2x": "1536px",
     },
+
+    // fontSize: {
+    //   xxs: "0.62rem",
+    //   xs: "0.75rem",
+    //   ms: "0.875rem",
+    //   base: "1rem",
+    //   lg: "1.125rem",
+    //   xl: "1.25 rem",
+    //   "2xl": "1.5rem",
+    //   "3xl": "1.85rem",
+    //   "4xl": "2.25rem",
+    //   "5xl": "3rem",
+    //   "6xl": "3.75 rem",
+    //   "7xl": "4.5rem",
+    //   "8xl": "6rem",
+    //   "9xl": "8rem",
```

Comment:
Not sure if we need this @JuliaZel @wu-ada? I couldn't find this specifications on our Figma. Could you guys confirm?

Reply:
Yes, good call, I believe we can get rid of this - @wu-ada can give final confirmation

Reply:
Yup! We don't need those. Thanks for catching that Nick!